calculatorlab/src/App.css

```diff
@@ -0,0 +1,38 @@
+body {
```

Comment:
incorporate flex or grid on to your css, this was asked as part of the rubric for this project.